src/functionaltests/java/com/ericsson/ei/utils/DataBaseManager.java

```diff
@@ -178,4 +183,19 @@ public int waitListSize() {
         List<Document> documents = table.find().into(new ArrayList<>());
         return documents.size();
     }
+
+    /**
+     * Get a specific subscription from the database based on the subscription name.
+     *
+     * @param subscriptionName
+     * @return the document as JSON string
+     */
+    public String getSubscription(String subscriptionName) {
+        mongoClient = new MongoClient(getMongoDbHost(), getMongoDbPort());
+        MongoDatabase db = mongoClient.getDatabase(database);
```

Comment:
Will not get a null point acception here if connection to database fail?
Shouldn't it be a try catch of if statetment that check the mongClient object before using object.

Reply:
And log the failure.

Reply:
It doesn't throw any exception. This apparently how it's done in similar methods in the same class. It's a helper class that connects to a local mongodb and only in functionaltest.

Reply:
Please leave it as is, When merged with changes for multiple MongoDB this has to be updated, keep it like it is then it will be simple.